### PR TITLE
Remove pages for AddressErrors

### PR DESCRIPTION
**Pull request by Elchi3**
Removed from the spec in https://github.com/w3c/payment-request/pull/955

There are no compat tables on these pages. Probably it wasn't supported anywhere.

### CODE REVIEW

**Comment by Elchi3** [comment]
Okay the large change to wiki history was automatic. Not sure why it did that.

File: files/en-us/_redirects.txt [hunk 1]
```diff
@@ -7342,8 +7342,6 @@
 /en-US/docs/Web/API/AbstractWorker	/en-US/docs/Web/API/Worker
 /en-US/docs/Web/API/AbstractWorker.onerror	/en-US/docs/Web/API/Worker/error_event
 /en-US/docs/Web/API/AbstractWorker/onerror	/en-US/docs/Web/API/Worker/error_event
-/en-US/docs/Web/API/AddressErrors/languageCode	/en-US/docs/Web/API/AddressErrors
-/en-US/docs/Web/API/AddressErrors/regionCode	/en-US/docs/Web/API/AddressErrors
```
Comment (bsmth) [inline files/en-us/_redirects.txt:7346]: Do we need redirects for these also?

* Web/API/AddressErrors/addressLine
* Web/API/AddressErrors/city
* Web/API/AddressErrors/country
* Web/API/AddressErrors/dependentLocality
* Web/API/AddressErrors
* Web/API/AddressErrors/organization
* Web/API/AddressErrors/phone
* Web/API/AddressErrors/postalCode
* Web/API/AddressErrors/recipient
* Web/API/AddressErrors/region
* Web/API/AddressErrors/sortingCode

Comment (Elchi3) [reply files/en-us/_redirects.txt:7346]: Not sure where these should redirect to. I don't think there is an alternative API.

Comment (bsmth) [reply files/en-us/_redirects.txt:7346]: Ah, I misread the changes as additions to the redirects, not deletions. Disregard the comment above

**Comment by Elchi3** [comment]
> Okay the large change to wiki history was automatic. Not sure why it did that.

https://github.com/mdn/content/pull/19935 apparently introduced linting to the file. How do I lint and why does it clash with the CLI command `yarn content delete <file>`?

**Comment by bsmth** [comment]
> Okay the large change to wiki history was automatic. Not sure why it did that.

How did that happen? Did CI do this? Can you reset the changes on your branch and make another commit?

**Comment by Elchi3** [comment]
ah thanks, waiting for https://github.com/mdn/yari/pull/7032 then. Marked as draft.

**Comment by bsmth** [comment]
> ah thanks, waiting for [mdn/yari#7032](https://github.com/mdn/yari/pull/7032) then. Marked as draft.

Very good, thanks

**Comment by Elchi3** [comment]
Alright, JSON reformatting solved. Ready to review.

**Review by bsmth — APPROVED** [review]
Looks good, thanks 👍🏻

**Pull request merged**